mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/QuickMarcUpdateKafkaHandler.java

```diff
@@ -44,32 +43,30 @@
   private static final String UNZIP_ERROR_MESSAGE = "Error during unzip";
   private static final String EVENT_ID_PREFIX = QuickMarcUpdateKafkaHandler.class.getSimpleName();
 
-  private final KafkaInternalCache kafkaInternalCache;
+  private final EventProcessedService eventProcessedService;
```

Comment:
There is no need for using this service. We can just remove the Kafka cache and rely on the database and handler that we have in mod-quick-marc.